src/cursor/cur_layered.c

```diff
@@ -842,7 +842,6 @@ __clayered_next(WT_CURSOR *cursor)
         goto retry;
 
     WT_STAT_CONN_DSRC_INCR(session, layered_curs_next);
-    /* FIXME-WT-15543: Handle the case of current_cursor being NULL */
```

Comment:
Sorry, I've left the wrong FIXME here, it should be [WT-15545](https://jira.mongodb.org/browse/WT-15545)

Let me know if you want to fix it yourself or you want me to fix it

Reply:
All good @ivankochin thank you.